src/adagrams.js

```diff
@@ -1,15 +1,97 @@
+// Implement this method for wave 1
+
+
 export const drawLetters = () => {
```

Comment:
⚠️ This has a subtle issue.  It gives you an equal chance to draw a Z and an A despite there being 9 As and only 1 Z.